apps/api/src/domain/models/user-management/user/commands/register-user.command.ts

```diff
@@ -5,7 +5,8 @@ import { ICreateCommand } from '../../../shared/command-handlers/interfaces/crea
 
 @Command({
     type: 'REGISTER_USER',
-    description: 'Registers a new user',
+    label: 'Register User',
+    description: 'Registers a user that already exists with the auth provider',
```

Comment:
Reword this. We're not registering it with the auth provider. We're registering it with our system. It already exists with the auth provider.

Reply:
done